### PR TITLE
Add ability to delete featured researchers

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

File: spec/controllers/content_blocks_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        r.save
+        u
+      end
+
+      before do
+        sign_in(user) if user
+      end
+
+      it "should delete a featured_researcher" do
+
```
Comment (wickr) [inline spec/controllers/content_blocks_controller_spec.rb]: Can you remove the extra linebreaks?

File: app/controllers/content_blocks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  def index
+  end
+
+  def destroy
+    @researcher = ContentBlock.find(params[:id])
+    if @researcher.destroy
+      flash[:success] = "Sucessfully deleted."
+    else
+      flash[:error] = "Error in deleting entry."
```
Comment (wickr) [inline app/controllers/content_blocks_controller.rb]: I think it'd be good to be a little more specific with the error, maybe "Error while deleting Featured Researcher."

File: circle.yml [hunk 3]
```diff
@@ -9,7 +9,6 @@ dependencies:
   pre:
     - gem install fcrepo_wrapper
     - gem install solr_wrapper
-    - npm install phantomjs@1.9.8
```
Comment (revgum) [inline circle.yml]: Think we want this back in the mix, right?

File: app/controllers/content_blocks_controller.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
+class ContentBlocksController < ApplicationController
```
Comment (revgum) [inline app/controllers/content_blocks_controller.rb]: Definitely extend Sufia's controller, that'll make the test and updates easier

File: circle.yml (hunk 3 above)
Comment (straleyb) [inline circle.yml]: Im not too sure. The tests seemed to pass without it. Im curious to see if they will keep passing. Ill add it back in, but if we can eventually take it out, that would be awesome.

**Comment by vantuyls** [comment]
thumbs up.

**Pull request merged**